src/Components/Common/PageTitle.tsx

```diff
@@ -43,6 +43,7 @@ export default function PageTitle(props: PageTitleProps) {
     if (divRef.current && focusOnLoad) {
       divRef.current.scrollIntoView({ behavior: "smooth" });
     }
+    console.log("Page title");
```

Comment:
@Pragati1610 I suppose you forgot to remove this.

Reply:
noted